web/src/user_profile.ts

```diff
@@ -687,23 +783,28 @@ export function show_edit_bot_info_modal(user_id, $container) {
         });
     });
 
-    function edit_bot_post_render() {
+    function edit_bot_post_render(): void {
         $("#edit_bot_modal .dialog_submit_button").prop("disabled", true);
 
-        function get_options() {
+        function get_options(): {
+            name: string;
+            unique_id: number;
+        }[] {
+            assert(bot?.is_bot);
             const user_ids = people.get_realm_active_human_user_ids();
-            if (!user_ids.includes(bot.bot_owner_id)) {
+            if (!user_ids.includes(bot.bot_owner_id!)) {
                 // Always include the current bot owner in
                 // options, even if the owner is deactivated.
-                user_ids.push(bot.bot_owner_id);
+                user_ids.push(bot.bot_owner_id!);
```

Comment:
The `bot_owner_id` logic is pretty clear in https://zulip.com/api/register-queue that `bot_owner_id` can be `null` for a bot.

So I think these `!` assertions are not correct; you should instead write `if (bot.bot_owner_id !== null && ...`)

Reply:
Yes, that was a bug. Fixed it!